src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -291,22 +291,28 @@ Plugin::Plugin()
               const auto specifiedDeviceName = get_specified_device_name(config);
               return _metrics->GetFullDeviceName(specifiedDeviceName);
           }}},
+        {ov::hint::model_priority.name(),
```

Comment:
`ov::hint::model_priority` is a property of CompiledModel, why do we need to have it listed also as a plugin property?

Reply:
Ah, yes, that makes sense. I see GPU plugin also includes `ov::hint::model_priority` as a supported option in the plugin

https://github.com/openvinotoolkit/openvino/blob/e6dd340dcaf01033eabeafec1091fe415dceb204/src/plugins/intel_gpu/src/plugin/plugin.cpp#L541

So it should be fine.